src/main/java/com/conveyal/gtfs/api/graphql/StopFetcher.java

```diff
@@ -62,7 +69,8 @@
                             .forEach(stops::add);
                 }
                 // get stops by bounding box
-                else if (args.get("max_lat") != null && args.get("max_lon") != null && args.get("min_lat") != null && args.get("min_lon") != null) {
+                else if (argumentDefined(env, "min_lat") && argumentDefined(env, "max_lat") &&
+                        argumentDefined(env, "min_lon") && argumentDefined(env, "max_lon")) {
                     Coordinate maxCoordinate = new Coordinate((Double) args.get("max_lon"), (Double) args.get("max_lat"));
```

Comment:
Good general rule of thumb is to check if maxLon > minLon and swap them if not, same for lat.